include/eve/algo/concepts/iterator_cardinal.hpp

```diff
@@ -55,17 +69,33 @@ namespace eve::algo
   template <typename I>
   constexpr std::ptrdiff_t iterator_cardinal_v = iterator_cardinal_t<I>{}();
 
-
   //================================================================================================
-  //! @addtogroup eve.algo.concepts
+  //! @addtogroup algo_concepts
   //! @{
-  //!  @struct wide_value_type
-  //!  @brief for an instance of `eve::algo::iterator` a shortcut: wide<value_type_t<I>, iterator_cardinal_t<I>>
+  //!   @struct wide_value_type
+  //!   @brief Computes an eve::simd_value from an iterator
```

Comment:
This is a bit confusing. But Ok.
We just have `iterator` and `relaxed_iterator`, where `relaxed_iterator` does not model `iterator`.
This is only designed for `iterator`. Maybe we should rename `iterator`.